packages/web-sdk/src/instrumentations/session/sessionManager/index.ts

```diff
@@ -1,4 +1,7 @@
 export { PersistentSessionsManager } from './PersistentSessionsManager';
 export { VolatileSessionsManager } from './VolatileSessionManager';
+
 export * from './sessionConstants';
```

Comment:
Nitpicky: I know it's a little cumbersome to write the exports again and again but usually it's best to avoid exporting everything.